### PR TITLE
feat: [#175488816] change payment label to wallet

### PR DESCRIPTION
**Pull request by debiff**
## Short description
This PR rename every occurrence of the Payment section in Wallet

## List of changes proposed in this pull request
- Renamed menu item
- Renamed title section
- Renamed occurrence in BONUS_REQUEST_INFORMATION
- Renamed occurrence in MESSAGE_DETAILS
- Renamed occurrence in BONUS_ACTIVATION_EXISTS

In the contextual help:
- Renamed occurrence in WALLET_HOME
- Renamed occurrence in WALLET_TRANSACTION_DETAILS
- Renamed occurrence in WALLET_LIST
- Renamed occurrence in WALLET_CARD_TRANSACTION
- Renamed occurrence in WALLET_ADD_PAYMENT_METHOD
- Renamed occurrence in WALLET_ADD_CARD
- Renamed occurrence in WALLET_CONFIRM_CARD_DETAILS
- Renamed occurrence in BONUS_ACTIVE_DETAIL_SCREEN
- Renamed occurrence in BONUS_AVAILABLE_LIST

### CODE REVIEW

File: ts/components/screens/BaseScreenComponent.tsx [hunk 1]
```diff
@@ -170,6 +176,20 @@ class BaseScreenComponent extends React.PureComponent<Props, State> {
     customHandledLink.map(link => handleItemOnPress(link)());
   };
 
+  private chMarkdown = (chTitle: string, chBody: string) => ({
```
Comment (Undermaken) [inline ts/components/screens/BaseScreenComponent.tsx:179]: Is this change needed?

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: ts/components/screens/BaseScreenComponent.tsx (hunk 1 above)
Comment (debiff) [reply ts/components/screens/BaseScreenComponent.tsx:179]: Not anymore after this PR https://github.com/pagopa/io-app/pull/2363

**Review by Undermaken — APPROVED** [review]
